resourcematch-api/main.py

```diff
@@ -148,11 +161,21 @@ def search_responses_to_urls(search_responses):
             page_uuid_partition = page_uuid.partition('@')[0]
             page_data = get_book_page_data(book_uuid, page_uuid_partition)
             page_url_main = page_data['urls']['main']
+            page_book_title = decode_html_entities(page_data['book']['title'])
+            page_chapter_title = decode_html_entities(
+                                 page_data['contextTitles'][1])
+            page_title = decode_html_entities(page_data['title'])
```

Comment:
Perhaps PDM/UX question: Would it be useful to use the title with enumeration (this is the one that will align with the page slug). For example, using either `page_data['titleParts']['title']` or `page_data['contextTitles'][-1]` (both of these should be the same I believe).